lib/pymedphys/_dicom/create.py

```diff
@@ -40,13 +40,17 @@ def set_default_transfer_syntax(dataset):
         dataset.is_implicit_VR = True
 
 
-def dicom_dataset_from_dict(input_dict: dict, template_ds=None):
+def dicom_dataset_from_dict(input_dict: dict, ensure_file_meta=False, template_ds=None):
```

Comment:
I was getting complaints of lacking File Meta when trying to write tests with skeleton DICOM datasets generated from `dict`. This fixed it.